src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterSuggestTests.cs

```diff
@@ -203,15 +207,19 @@ public void TestSuggestMutationFunctions(string expression, params string[] expe
 
         [InlineData("Collect(|, record1", "Entity1", "Entity2", "Table1", "table2")]
         [InlineData("Sum(|", "num", "str")]
+        [InlineData("Sum(nu|", "Minute", "num", "IsNumeric")]
         [InlineData("Text(|", "num", "str")]
         [InlineData("Language(|")]
         [InlineData("Filter([1,2], |", "ThisRecord", "Value")]
 
         // Suggests Enum.
         [InlineData("Text(Now(), |", "DateTimeFormat.LongDate", "DateTimeFormat.LongDateTime", "DateTimeFormat.LongDateTime24", "DateTimeFormat.LongTime", "DateTimeFormat.LongTime24", "DateTimeFormat.ShortDate", "DateTimeFormat.ShortDateTime", "DateTimeFormat.ShortDateTime24", "DateTimeFormat.ShortTime", "DateTimeFormat.ShortTime24", "DateTimeFormat.UTC", "num", "str")]
+        [InlineData("DateDiff(Date(2023,1,1), Date(2023, 2,1), |", "TimeUnit.Days", "TimeUnit.Hours", "TimeUnit.Milliseconds", "TimeUnit.Minutes", "TimeUnit.Months", "TimeUnit.Quarters", "TimeUnit.Seconds", "TimeUnit.Years")]
+        [InlineData("DateDiff(Date(2023,1,1), Date(2023, 2,1), T|", "Not", "TimeUnit.Days", "TimeUnit.Hours", "TimeUnit.Milliseconds", "TimeUnit.Minutes", "TimeUnit.Months", "TimeUnit.Quarters", "TimeUnit.Seconds", "TimeUnit.Years", "true")]
 
         // Custom Function has arg with signature of tableType2, So only suggest table2
         [InlineData("RecordsTest(|", "table2")]
+        [InlineData("RecordsTest(t|", "table2", "Distinct", "Filter", "Not", "Sort",  "true")]
```

Comment:
>true [](http://example.com/codeflow?start=87&length=4)

Do we really want to exclude keyword? today is slightly trick because there is no inherent type mapping to a keyword (could be done though if needed)